src/php/utils/install-shush

```diff
@@ -2,7 +2,9 @@
 
 set -xe
 
-curl -sL -o /usr/local/bin/shush https://github.com/realestate-com-au/shush/releases/download/v1.3.4/shush_linux_amd64
+curl -sL -o /usr/local/bin/shush https://github.com/realestate-com-au/shush/releases/download/v1.5.0/shush_linux_amd64
+
+echo "cdec941dc5f45dda2d981169aa1845540d2c5bf98bfd1d8a85deaa6a6a43a4d1  /usr/local/bin/shush" | sha256sum -c
```

Comment:
From where this `sha` comes from? Should we add it to the commit message, so in the future we know where to look for?

Reply:
The checksum of a file can be computed by running the command below:
```bash
$ sha256sum <file_name>
```
I thought this was common knowledge :shrug:
If you like I add a reference to [the manpage](https://man7.org/linux/man-pages/man1/sha256sum.1.html).